blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,92 @@
+
+What Is extension-manager?
+
+Extension-manager is a key component that handles lifecycle operations (create, upgrade, delete, reconcile) for Kubernetes extensions. It periodically polls for version updates on extension instances and reconciles extension configuration settings, ensuring that extensions are managed effectively. Learn more about the flow in [extension architecture](https://learn.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions#architecture). 
+
+Previously, extension-manager pods (extension-operator and extension-agent) were deployed in the kube-system namespace on customer worker nodes. Example of a deployment before:
+```bash
+$ kubectl get pods -n kube-system
+NAME                                      READY   STATUS    RESTARTS   AGE
+coredns-558bd4d5db-7x5c8                  1/1     Running   0          5d
+extension-agent-abc123                    1/1     Running   0          3d
+extension-operator-def456                 1/1     Running   0          3d
+```
+
+What Has Changed?
+
+Extension-manager components are now moved to the AKS control plane hosted on Microsoft tenant and subscriptions. Extension-manager is no longer visible to users.
+
+Benefits of the Move
+1. **Reduced Identity Footprint:** Eliminates the need for node-level identities for extension-manager, improving security and reducing setup time. This also contributes to shortening the first extension installation time.
+1. **Reduced resource usage**. Moving the extension-manager pods to AKS internal infrastructure will reduce the resource usage on the customer's side.
+1. **Improved extension integrity:** Prevents accidental interference with extension components, safeguarding extension integrity.
+1. **Simplified Networking:** Reduces the need for complex networking configurations. Before the migration, users need to set up required networking rules as outlined in [Required FQDN / application rules](https://learn.microsoft.com/en-us/azure/aks/outbound-rules-control-egress#required-fqdn--application-rules-5) for extension-manager. Extension-manager now communicates directly with the following endpoints from AKS Control Plane, removing the need for users to configure them. Note that some extensions may still require users to configure networking for the extension instances on the users' clusters (e.g., allowing access to MCR for pulling container images).
+    * `<region>.dp.kubernetesconfiguration.azure.com`
+    * `mcr.microsoft.com`, `*.data.mcr.microsoft.com`
+    * `arcmktplaceprod.azurecr.io` and `arcmktplaceprod.<region>.data.azurecr.io`
+
+## Which applications or services will be supported through core Kubernetes extensions?
+We're in the process of enabling Container insights, Managed prometheus, App monitoring, and Azure Backup through core extensions, and there will be more. Keep an eye out for more updates, and share your feedback with us!
```

Comment:
Do we have links we can put here for these? (I think you had some above, maybe proxy down here too?)